src/nashpy/integer_pivoting/integer_pivoting_lex.py

```diff
@@ -0,0 +1,108 @@
+"""
+A class for integer pivoting. Used for an implementation of the Lemke Howson
+algorithm.
+"""
+import numpy as np
+
+import warnings
+
+
+def make_tableau(M):
+    """
+    Make a tableau for the given matrix M.
+
+    This tableau corresponds to the polytope of the form:
+
+       Mx <= 1 and x >= 0
+    """
+    return np.append(
+        np.append(M, np.eye(M.shape[0]), axis=1),
+        np.ones((M.shape[0], 1)),
+        axis=1,
+    )
+
+
+def find_pivot_row_lex(tableau, column_index, slack_variables):
+
+    """
+    Find the index of the row to pivot.
+
+    Identifies the row to pivot by finding the first fow in a lexicographical 
+    ordering of rows. (First checks minimum ratio test, then uses lexicographical
+     ordering to break ties)
```

Comment:
Could you add to the docstring please:

- References to text you're basing the implementation on;
- Explanation of variables `C` and `Cq` in particular but possibly more.

Reply:
I added some information here, hopefully it is enough to explain how the pivoting works.